core/src/main/java/org/apache/carbondata/common/iudprocessor/iuddata/DeleteDeltaDataUtil.java

```diff
@@ -0,0 +1,44 @@
+package org.apache.carbondata.common.iudprocessor.iuddata;
+
+import org.apache.carbondata.core.carbon.AbsoluteTableIdentifier;
+import org.apache.carbondata.core.carbon.metadata.CarbonMetadata;
+import org.apache.carbondata.core.carbon.metadata.schema.table.CarbonTable;
+import org.apache.carbondata.core.load.LoadMetadataDetails;
+import org.apache.carbondata.core.updatestatus.SegmentStatusManager;
+
+/**
+ * Created by S71955 on 06-10-2016.
+ */
+public class DeleteDeltaDataUtil {
+
+  private DeleteDeltaDataUtil() {
```

Comment:
Looks like created singleton class but not using  , pls check and remove this class